### DISCUSSION TITLE
Can I use react-native-firebase firestore and function in my expo project?

### DISCUSSION THREAD

**Discussion by cbdeveloper** [invertase/react-native-firebase#7277] [Q&A]
https://rnfirebase.io/#expo
From the docs:

> Config plugins are only required for React Native Firebase modules, which require custom native installation steps - e.g. modifying the Xcode project, Podfile, build.gradle, AndroidManifest.xml etc. Packages without native steps required will work out of the box.

> Not all React Native Firebase packages have Expo config plugins provided yet. You may see if a module is supported by checking if it contains the app.plugin.js file in its package directory.

I can see that both firestore and functions do not have the app.plugin.js file. Does that mean I cannot use them in my expo project?

Thank you

![image](https://github.com/invertase/react-native-firebase/assets/43407798/a51b7822-7149-4d1a-80fb-ff22dab098c9)

**Comment by haeniya** [comment]
@cbdeveloper have you tried it since? According to my research it should be possible to use firestore in Expo. But not sure about the functions and storage packages.

**Reply by cbdeveloper** [reply]
No, I haven't tried since.

**Comment by erdincakdoganmoi** [comment]
Yes you can use it my friend.  You can find lots of videos about this
